cn_indian_payroll/cn_indian_payroll/overrides/payroll_configuration.py

```
@@ -1,119 +1,242 @@
+
+
+        if int(component) == 1 and int(custom_field) == 1:
+            if salary_component:
+
+                get_salary_component = frappe.get_list('Salary Component',
                 filters={
-                    'dt': 'Salary Structure Assignment',
-                    'fieldname': assignment.field_name,
+                    "name":salary_component,
+                    "disabled":0,
+                    "type":component_type
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)